src/adapter/graphql/src/queries/admin/admin.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::prelude::*;
+use crate::AdminGuard;
+
+pub struct Admin;
+
+#[Object]
+impl Admin {
+    #[graphql(guard = "AdminGuard::new()")]
+    async fn self_test(&self, _ctx: &Context<'_>) -> Result<String> {
+        Ok("OK".to_string())
+    }
```

Comment:
It might be a nice demo for how to use the new guard class, but I also expect a very simple boolean attribute on the `Account` schema.

Reply:
Done via https://github.com/kamu-data/kamu-cli/pull/414/commits/0e7bec378591a878b423ae3cd08c9ba43e0234d3

<details>
  <summary>Example request/response pair</summary>

Request:
```graphql
query {
    accounts {
        byName(name: "s373r") {
            displayName
            isAdmin
            id
            accountName
            displayName
            avatarUrl
        }
    }
}
```

Response:
```json
{
  "data": {
    "accounts": {
      "byName": {
        "displayName": "Dima Pristupa",
        "isAdmin": true,
        "id": "1337",
        "accountName": "s373r",
        "avatarUrl": "https://avatars.githubusercontent.com/u/50896974?s=200&v=4"
      }
    }
  },
  "extensions": {}
}
```
</details>